db/python/layers/search.py

```diff
@@ -103,27 +157,62 @@ async def search(self, query: str, project_ids: list[int]) -> List[SearchRespons
             )
 
             return [response] if response else []
+    
+        if cpg_sg_id := self.try_get_sg_id_from_query(query):
+            # just get the sg
+            response = await self._get_search_result_for_sequencing_group(
+                cpg_sg_id, project_ids=project_ids
+            )
+            print(response)
+
+            return [response] if response else []
 
         ftable = FamilyTable(self.connection)
         ptable = ParticipantTable(self.connection)
         stable = SampleTable(self.connection)
+        sgtable = SequencingGroupTable(self.connection)
 
         sample_rows, participant_rows, family_rows = await asyncio.gather(
             stable.search(query, project_ids=project_ids, limit=5),
             ptable.search(query, project_ids=project_ids, limit=5),
             ftable.search(query, project_ids=project_ids, limit=5),
+            # sgtable.search(query, project_ids=project_ids, limit=5),
         )
+        # print(sg_rows)
 
         sample_participant_ids = [s[2] for s in sample_rows]
         all_participant_ids = list(
             set(sample_participant_ids + [p[1] for p in participant_rows])
         )
+        # seq_group_participant_ids = [sg[3] for sg in sg_rows]
 
         sample_participant_eids, participant_family_eids = await asyncio.gather(
             ptable.get_external_ids_by_participant(sample_participant_ids),
             ftable.get_family_external_ids_by_participant_ids(all_participant_ids),
+            # ptable.get_external_ids_by_participant(seq_group_participant_ids),
         )
 
+        # sequencing_groups = [
```

Comment:
Is this still supposed to be commented out?

Reply:
@illusional  I don't think the test data has SG external IDs, probably why I had such trouble writing the query. Should we merge the exact match SG internal ID search at least and add in the external ID string match later? I don't know how much use case there is for string matching on IDs - you'd probably need to know what you're searching for before you search it, and if you're just exploring the data then the project grid is more useful for that?

Reply:
Yeah that seems reasonable, can you add a TODO in the spot where we should add the SG external ID searching